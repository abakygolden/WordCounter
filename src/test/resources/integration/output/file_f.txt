F 1
FACE 1
FAMES 1
FANTAS 1
FAR 1
FAUCIBUS 2
FEJL 1
FERMENTUM 2
FIERRO 1
FINDS 1
FINIBUS 1
FJENDERS 1
FL 1
FLOWER 1
FODBOLDPUBLIKUM 1
FOLK 1
FOLLOWING 1
FORDUMS 1
FORHOLD 1
FORSKELLIGARTET 1
FRA 3
FREJAS 2
FREM 1
FREMMEDE 1
FRIENDS 1
FRINGILLA 2
FRIT 1
FRODO 1
FROST 1