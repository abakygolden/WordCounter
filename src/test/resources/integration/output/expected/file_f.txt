FAJITA 4
FAJITAS 4
FALAFEL 8
FEIJOA 4
FETTUCCINE 4
FIG 4
FINGER 4
FISH 4
FONDUE 4
FRENCH 4
FRIED 12
FRUIT 16
FRY 8